### DISCUSSION TITLE
Please advise.

### DISCUSSION THREAD

**Discussion by ilfae** [ciaochaos/qrbtf#54] [Q&A]
Please suggest a promt to do the same: [image
](https://media.discordapp.net/attachments/1117758064175628288/1119222608332853248/photo_2023-06-06_20-16-36.jpg?width=683&height=683)
What kind of promt was used for this?

**Comment by GerardPieterse** [comment]
I believe they developed a new control net model made for QR, so you cant get results like these without it.

**Reply by ilfae** [reply]
Got it, thanks.
